benchmark/ubench.c

```diff
@@ -238,8 +239,10 @@ UBENCH_EX(simple, disjoint_pool_with_os_memory_provider) {
     disjoint_memory_pool_params.MinBucketSize = DISJOINT_POOL_MIN_BUCKET_SIZE;
 
     umf_memory_pool_handle_t disjoint_pool;
-    umf_result = umfPoolCreate(umfDisjointPoolOps(), os_memory_provider,
-                               &disjoint_memory_pool_params, 0, &disjoint_pool);
+    // disable the tracking provider to make it work faster (UMF_POOL_CREATE_FLAG_DISABLE_TRACKING)
```

Comment:
Are we interested in performance without a tracking provider? But in real workloads tracking capabilities will be ON?

Reply:
Right. Removed.
Done.